src/libraries/Microsoft.PowerFx.Core/Functions/ArgPreprocessor.cs

```diff
@@ -9,9 +9,16 @@ namespace Microsoft.PowerFx.Core.Functions
 {
     internal enum ArgPreprocessor
     {
-        None = 0,
-        ReplaceBlankWithZero = 1,
-        ReplaceBlankWithZeroAndTruncate = 2,
-        ReplaceBlankWithEmptyString = 3,
+        None = 0,
+
+        ReplaceBlankWithFloatZero = 1,
+        ReplaceBlankWithDecimalZero = 2,
+        ReplaceBlankWithFloatZeroAndTruncate = 3,
```

Comment:
does this one need a decimal equivalent?

Reply:
Added.  It isn't used today, but probably wise to include it so someone has it when they need it.